components/popover_list_members/popover_list_members.jsx

```diff
@@ -124,9 +124,9 @@ export default class PopoverListMembers extends React.Component {
                             key={'popover-member-' + i}
                         >
                             <ProfilePicture
+                                className='user__picture'
                                 src={Client4.getProfilePictureUrl(m.id, m.last_picture_update)}
-                                width='40'
-                                height='40'
+                                status={UserStore.getStatus(m.id)}
```

Comment:
Instead of contacting the store directly, can you pass this in from the container? You can use the `getStatusForUserId` selector from `mattermost-redux/selectors/entities/users` to get it from the redux store instead of using the old `UserStore`

Reply:
I was using the store because all the status example around the code were also using the store. I am not sure if I did it right as I am not knowledge in redux, but let me know, if the syntax is right.